assets/js/templates/keyboard-shortcuts-help-modal.handlebars

```diff
@@ -0,0 +1,25 @@
+<div id="keyboard-shortcuts-modal-wrapper">
+  <div id="keyboard-shortcuts-modal" tabindex="-1">
+    <div class="modal-contents">
+      <div class="modal-header">
+        <div class="modal-title">Keyboard Shortcuts</div>
+        <div class="modal-close">×</div>
+      </div>
+      <div class="modal-body">
+        {{#each shortcuts}}
+          <dl class="shortcut-row">
+            <dt class="shortcut-keys">
+              {{#if this.requiresShiftKey}}
+                <div class="shortcut-key">shift</div>
```

Comment:
what do you think about using `<kbd>shift</kbd>` etc for keys, like GitHub does?